### PR TITLE
Send player info

### PR DESCRIPTION
**Pull request by karpov-kir**
Related to: https://bitmovin.atlassian.net/browse/TA-2598.

Conviva now has an API to send player info. Integrate it and cover it with tests.

### CODE REVIEW

File: example/index.html [hunk 1]
```diff
@@ -77,13 +77,10 @@
           custom: {
             appVersion: '1.0',
             contentId: 'someContentId',
-            playerVendor: 'bitmovin',
-            playerVersion: player.version,
```
Comment (karpov-kir) [inline example/index.html:81]: Removed to avoid confusion

File: spec/tests/ConvivaAnalytics.spec.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    convivaAnalytics = new ConvivaAnalytics(playerMock, 'TEST-KEY');
+  });
+
+  describe('when initializing session', () => {
+    it('set player info', () => {
+      playerMock.eventEmitter.firePlayEvent();
+
+      expect(convivaVideoAnalytics.setPlayerInfo).toHaveBeenLastCalledWith(
+        expect.objectContaining({
+          frameworkName: 'bitdash',
```
Comment (dweinber) [inline spec/tests/ConvivaAnalytics.spec.ts]: We don't use `bitdash` publicly (anymore). Please change to `Bitmovin Player`.

```suggestion
          frameworkName: 'Bitmovin Player',
```

File: src/ts/ConvivaAnalytics.ts [hunk 3]
```diff
@@ -443,12 +443,19 @@ export class ConvivaAnalytics {
 
     // Create a Conviva monitoring session.
     this.convivaVideoAnalytics = Conviva.Analytics.buildVideoAnalytics();
+
+    this.convivaVideoAnalytics.setPlayerInfo({
+      [Conviva.Constants.FRAMEWORK_NAME]: 'bitdash',
```
Comment (dweinber) [inline src/ts/ConvivaAnalytics.ts]: We don't use `bitdash` publicly (anymore). Please change to `Bitmovin Player`.

```suggestion
      [Conviva.Constants.FRAMEWORK_NAME]: 'Bitmovin Player',
```

**Review by dweinber — CHANGES_REQUESTED** [review]
Nice addition. Please change the player name though, thanks.

**Review by dweinber — APPROVED** [review]

**Pull request merged**